bin/tbg

```diff
@@ -208,15 +207,15 @@ function check_final
     final_file="$1"
     org_file="$2"
 
-    not_replaced=`grep -o "\![[:alpha:]][[:alnum:]_]*" $final_file | sort | uniq`
-    not_replaced_cnt=`echo $not_replaced | wc -w`
+    not_replaced=$(grep -o "\![[:alpha:]][[:alnum:]_]*" $final_file | sort | uniq)
+    not_replaced_cnt=$(echo $not_replaced | wc -w)
 
     if [ $not_replaced_cnt -gt 0 ] ; then
         echo "ERROR: $not_replaced_cnt variable(s) _not_ replaced from template (tpl):"
         echo $not_replaced
 
         #create an OR concated pattern
-        nrv_or=`echo $not_replaced | sed 's/[[:space:]]/|/g'`
+        nrv_or=$(echo $not_replaced | sed 's/[[:space:]]/|/g')
```

Comment:
That's most likely the line causing the warning

Reply:
I fixed it. It was coming from `egrep` not sed. I do not fully understand why `[[:blank:]]` is throwing the warning.
I switched to the old-school way of defining a list of characters.

Reply:
https://github.com/ComputationalRadiationPhysics/picongpu/pull/4906/files#diff-7b05b982bf71e63ce4dd3f2fd29332ae3e3023f0357d503da6d0f011d936a155R393 is the problematic line

Reply:
@PrometheusPi this is already solved in the PR

Reply:
I checked that I used your (new?) version with `nrv_or=$(echo $not_replaced | sed 's/[[:space:]]/|/g')` but I still get the warning:
```
grep: character class syntax is [[:space:]], not [:space:]
```

Reply:
offline checked: @PrometheusPi did a mistake and used the old version and not the latest PR.

Reply:
Yes, I pulled something from GitHib, bit not the newest version. Tested it now and it does not show the warning anymore.